tests/functional/tests/management/test_start_stop.py

```diff
@@ -330,6 +330,21 @@ def test_start_cache_same_id(pyocf_ctx, mode, cls):
     cache.get_stats()
 
 
+def test_start_cache_huge_device(pyocf_ctx):
```

Comment:
You didn't give the description for this test.
Also maybe You want to try parametrize it with a `cache line size`?

Reply:
Added